src/pytroll_watchers/selector.py

```diff
@@ -0,0 +1,214 @@
+"""Functions and classes for performing message selection.
+
+Selection in this context means making sure only one message refering to some file will be published further.
+
+This is useful when multiple source for the same data are sending messages (eg two reception servers for eumetcast) but
+only one of each file is needed for further processing.
+
+At the moment, this module makes use of redis as a refined dictionary for keeping track of the received files. Hence a
+redis server instance will be started along with some of the functions here, and with the cli.
```

Comment:
Is the Redis database persisted? Where is it stored? If the database is not persisted, would a more lightweight solution (like `dict`) be an option?

Reply:
I haven't explored all of redis' possibilities, so I'm using the defaults now. The directory where the data is persisted can be defined through the `directory` parameter in the selector config (see docstrings).
I have looked for alternatives such as a dict with ttl for items but I haven't found anything that would work really.